en/docs/getting-started/about-this-release.md

```diff
@@ -4,33 +4,70 @@ WSO2 API Microgateway 3.2.0 is the successor of WSO2 API Microgateway version 3.
 
 -   [What's New](#whats-new)
 -   [What's Improved](#whats-improved)
--   [What has changed in this release](#what-has-changed-in-this-release)
 -   [Fixed issues](#fixed-issues)
 -   [Known issues](#known-issues)
 
 ### What's New
 
--  **Circuit Breaker, Retry, and Timeout Capabilities**   
-   WSO2 API Microgateway 3.2.0 is capable of applying circuit breakers to services in order to avoid any malfunctioning that can occur due to excess requests reaching a service from an overload. It also supports retries and timeouts to a service.
-
-- **JWKS support for JWT verification**
-  JWKS endpoint can now be configured for a JWT issuer. This brings in the ability to perform JWT verification using different issuers.
-
-- **Custom Claims Mapping for JWTs**
- WSO2 API Microgateway 3.2.0 facilitates claim mapping for JWTs, particularly useful when the claims of the incoming JWT cannot be directly validated by the backend. Also, it brings in the flexibility to generate JWTs with preferred custom claims.
- 
-- **Support Custom Siddhi Policies and Deny Policies**
-  WSO2 Microgateway 3.2.0 supports custom Siddhi based throttling policies making it easier to enforce complex customized throttling policies for an API.
+-   **Circuit Breaker, Retry, and Timeout Capabilities**
```

Comment:
This change has resulted in below view. Need to push description below the topic as before
<img width="848" alt="image" src="https://user-images.githubusercontent.com/2655553/92682596-59ee2780-f34e-11ea-89a8-c36d3e261ad0.png">